app/helpers/security_helper.rb

```diff
@@ -19,13 +19,25 @@ def add_stream_cookies(stream_info)
     end
   end
 
-  def secure_streams(stream_info)
+  def secure_streams(stream_info, media_object_id)
+    begin
+      if Avalon::Configuration.controlled_digital_lending_enabled? && Checkout.checked_out_to_user(media_object_id, current_user.id).empty?
+        raise StreamToken::Unauthorized
+      else
+        add_stream_url(stream_info)
+      end
+    rescue NoMethodError
```

Comment:
There may be a better way to handle this, but this seems to allow all of the previously problematic tests to still pass and allows the item view page to load for unauthenticated users which is an issue I ran into when moving the logic I had put in for the hls_manifest case.